package Protos;

message StdpC {
    required double tau_plus = 1;
    required double tau_minus = 2;
    required double a_plus = 3;
    required double a_minus = 4;
    required double learning_rate = 5;
}

message StdpState {
    required double y = 1;
    repeated double x = 2;
}

